shared/actions/fs/index.js

```diff
@@ -536,7 +537,13 @@ function* _loadMimeType(path: Types.Path, refreshTag?: Types.RefreshTag) {
         // but the path has been removed since then.
         return
       }
-      throw err
+      // It's still possible we have a critical error, but if it's just the
+      // server port number that's changed, it's hard to detect. So just treat
+      // all other errors as this case. If this is actually a critical error,
+      // we end up doing this 3 times for nothing, which isn't the end of the
+      // world.
+      logger.info(`_loadMimeType i=${i} error:`, err)
+      localHTTPServerInfo.address = ''
```

Comment:
So this will trigger a re-establishment of the server?

Reply:
Hmm not sure what you mean by "re-establishment of the server", but on next iteration this causes a call to `SimpleFSSimpleFSGetHTTPAddressAndTokenRpcPromise`, which refreshes address (with port) and token.